### ISSUE TITLE
Add ioq3 to the demonstrator image

### ISSUE THREAD

**Issue by edugrasa** [IRATI/demonstrator#6]
To allow for more fancy demos

**Comment by edugrasa** [comment]
Updated by #7 , should be a different image.

**Comment by vmaffione** [comment]
Yes, should be a different image. I need to know the dependencies required by ioq3, maybe @sandervrijders can help?

**Comment by sandervrijders** [comment]
The dependencies are quite a lot. You need a working window manager with openGL support. You also need the SDL2 library for sound. Don't know if that is feasible on a buildroot image. Also running Quake in a VM is quite slow unless you have a very good setup.

**Comment by edugrasa** [comment]
Maybe running a quake client is too much, but running the quake server
would be nice though (doesn't have graphical dependencies).

2016-07-08 9:58 GMT+02:00 Sander Vrijders notifications@github.com:

**Comment by vmaffione** [comment]
yep, the client is probably a no-go.. but we could try with the server. Can we come up with a complete list of dependencies? E.g. Archlinux or Ubuntu packages?

**Comment by dstaesse** [comment]
Is that really useful?

**Comment by edugrasa** [comment]
Yes, to be able to do demos like the one we did in TNC using the demonstrator.
And for people to play with it as well. I mean, if the effort is not that big (doesn't look like?)
